README.md

```diff
@@ -1,237 +1,109 @@
-# byu-jwt
+# Packages
 
-This package provides helpful functions for using validating and using BYU's JWTs.
+This repository hosts the following Node.js packages for API development at BYU OIT:
 
-[![codecov](https://codecov.io/gh/byu-oit/byu-jwt-nodejs/branch/main/graph/badge.svg?token=zOI4URNx3D)](https://codecov.io/gh/byu-oit/byu-jwt-nodejs)
+- [@byu-oit/jwt](./packages/jwt/README.md)
+- [@byu-oit/fastify-jwt](./packages/fastify/README.md)
 
-**Requires Node 8 or above**
+The documentation and source code for previous versions of the byu-jwt package are found on
+the [v3 branch](https://github.com/byu-oit/byu-jwt-nodejs/tree/v3) in this repository.
```

Comment:
Thanks for cleaning up the README. At some point, perhaps after go-live of these new packages, can we create something (a pinned issue, maybe?) to help people migrate?

Reply:
Correct, a migration guide is necessary for this change.